.azure-pipelines/pipelines.yml

```diff
@@ -137,6 +137,31 @@ jobs:
           DOCKERHUB_USERNAME: $(DockerUsername)
           DOCKERHUB_PASSWORD: $(DockerPassword)
 
+  - job: docker_arm64
+    displayName: "Linux-arm64 docker"
+    dependsOn: ["release_arm64"]
+    condition: and(succeeded(), eq(variables['PostSubmit'], 'true'), ne(variables['Build.Reason'], 'PullRequest'))
+    pool: "arm-large"
```

Comment:
Can we do this in existing docker job with multiarch?

Reply:
Hi, lizan
Yes, we can. And the buildx tool could make code more clean. But, the build image by qemu may be a little bit slow, i don't know whether there is enough time to do this. So I choose to build it on the arm environment.

Reply:
What is your opinion about it?

Reply:
I'm leaning to do it with buildx, I don't think you'll face any issue, we just do base image + putting binary in it.

Reply:
Sound good! I will cancel this PR.

Reply:
Hi, I update this patch by buildx tool.

Reply:
Hi, Could you please give some comments @lizan